follower.py

```diff
@@ -0,0 +1,186 @@
+        #the list with the new follower X  marked in it will be put back into the csv file
+
+
+# returns a lsit with all the pictures they posted
+def imagesForUser(user):
+    imagelist = []
+    with open('data/userimages.csv', newline='') as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='|')
+        imagelist = list(reader)
+        # opens the csv file with the images and checks 
+
+    userExist = False
+    userIndex = 0
+    for index, people in enumerate(imagelist):
+        if people[0] == user:
+            userExist = True
+            userIndex = index
+    if  not userExist:
+        return 
+    # testing to make sure the user is real and takes the users position
+
+    # will return a list of all images the user has
+    return imagelist[userIndex][1:]
+
+# returns a list with all the followers of a specific user
+def getUserFollowers(user):
+    followerlist = []
+    with open('data/followers.csv', newline='') as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='|')
+        followerlist = list(reader)
+        # opens the csv file where the followers are stored and tracks the data inside a list
+    
+    userExist = False
+    for people in followerlist[0]:
+        if people == user:
+            userExist = True
+    if  not userExist:
+        return
+    # testing to make sure the user and followers are real
+
+    userIndex = followerlist[0].index(user)
+    followers = []
+    for x in range(len(followerlist[userIndex])):
+        if followerlist[userIndex][x] == "X":
+            followers.append(followerlist[x][0])
+            # checks the user to see all the people they follow and store it in a list
+    return followers
+
+
+# returns a list with all images to be displayed for a user
+def getImagesToShow(user):
+    imagelist = []
+    with open('data/userimages.csv', newline='') as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='|')
+        imagelist = list(reader)
+        # opens the csv file with the images and checks 
+
+    userExist = False
+    for people in imagelist:
+        if people[0] == user:
+            userExist = True
+    if  not userExist:
+        return 
+        # testing to make sure the user is real and takes the users position
+    
+    
+    followerlist = getUserFollowers(user)
+    picturesDisplay = []
+    for follower in followerlist:
+        picturesDisplay= picturesDisplay + imagesForUser(follower)
+    return picturesDisplay
+    
+
+
+# testof the methods
+
+# addUser("Mikeyyy")
+# follow("Drand1943","Milloon")
+# follow("Drand1943","Drand1943")
+# follow("Drand1943","wgerwgg")
+# follow("sdfwergwghr","Drand1943")
+# print(imagesForUser("Giarturner"))
+# print(getUserFollowers("Cagoo1938"))
+# print(getImagesToShow("Whimseeplis"))
+
+
+# *************************************************************************DO NOT USE METHODS***************************************************************************
```

Comment:
Would commenting these methods out be useful at a later point? I see how they can still be useful at this point in the project (Sprint 2), so I am fine with them not being commented out currently.

Reply:
Yes, once we all discuss the databases I created and no more changes need to be made, these methods will be commented out. For now, we might need to use them so I left them available with a warning to make sure people do not use them without knowing what they are doing